ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,902 @@
+
+#. Run the Java producer to ``demo-topic-1`` after configuring the ACLs:
+
+   .. code-block:: bash
+
+      mvn -q -f ../../clients/cloud/java/pom.xml exec:java -Dexec.mainClass="io.confluent.examples.clients.cloud.ProducerExample" -Dexec.args="/tmp/client.config demo-topic-1" -Dlog4j.configuration=file:log4j.properties > /tmp/log.2 2>&1
+
+#. Verify you see the ``10 messages were produced to topic`` message in the logs as
+   shown in the following example:
+
+   .. code-block:: text
+
+         PASS
+         [2020-08-29 13:52:10,836] WARN The configuration 'sasl.jaas.config' was supplied but isn't a known config. (org.apache.kafka.clients.admin.AdminClientConfig)
+         [2020-08-29 13:52:10,837] WARN The configuration 'ssl.endpoint.identification.algorithm' was supplied but isn't a known config. (org.apache.kafka.clients.admin.AdminClientConfig)
+         Producing record: alice	{"count":0}
+         Producing record: alice	{"count":1}
+         Producing record: alice	{"count":2}
+         Producing record: alice	{"count":3}
+         Producing record: alice	{"count":4}
+         Producing record: alice	{"count":5}
+         Producing record: alice	{"count":6}
+         Producing record: alice	{"count":7}
+         Producing record: alice	{"count":8}
+         Producing record: alice	{"count":9}
+         Produced record to topic demo-topic-1 partition [3] @ offset 0
+         Produced record to topic demo-topic-1 partition [3] @ offset 1
+         Produced record to topic demo-topic-1 partition [3] @ offset 2
+         Produced record to topic demo-topic-1 partition [3] @ offset 3
+         Produced record to topic demo-topic-1 partition [3] @ offset 4
+         Produced record to topic demo-topic-1 partition [3] @ offset 5
+         Produced record to topic demo-topic-1 partition [3] @ offset 6
+         Produced record to topic demo-topic-1 partition [3] @ offset 7
+         Produced record to topic demo-topic-1 partition [3] @ offset 8
+         Produced record to topic demo-topic-1 partition [3] @ offset 9
+         10 messages were produced to topic demo-topic-1
+
+#. Run the following commands to delete the ACLs:
+
+   .. code-block:: bash
+
+      ccloud kafka acl delete --allow --service-account 104349 --operation CREATE --topic demo-topic-1
+      ccloud kafka acl delete --allow --service-account 104349 --operation WRITE --topic demo-topic-1
+
+   You should see a ``Deleted ACLs`` message.
+
+#. Create a new Kafka topic ``demo-topic-2``:
+
+   .. code-block:: bash
+
+      ccloud kafka topic create demo-topic-2
+
+   You should see a ``Created topic "demo-topic-2"`` message.
+
+.. shouldn't the following say "prefixed ACL" instead of "prefix ACL"
+
+Run a Java producer to showcase a prefix ACL
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Run the following command to create ACLs for the producer using a prefix:
+
+   .. code-block:: bash
+
```

Comment:
.. should the commands say "--prefixed" instead of "--prefix"

Reply:
The command itself is `--prefix`

Reply:
Okay cool